### DISCUSSION TITLE
How to Create PVC Longhorn-Static in YAML file ?

### DISCUSSION THREAD

**Discussion by northboys** [longhorn/longhorn#3101] [Q&A]
i trying to create pvc longhorn static on yaml file without on UI..could you give me an example ?

**Comment by jenting** [comment]
Not quite understand your question. But if you want to create a PVC, here is an [example](https://longhorn.io/docs/1.2.2/references/examples/#simple-persistentvolumeclaim).

The storage class points to use `lLonghorn`. If you want to use storage class `longhorn-static`, if I remember correctly, it's only available when creating PVC from GUI.
